### DISCUSSION TITLE
Is possible to insert other glb on WebXR ar-mode?

### DISCUSSION THREAD

**Discussion by GiovanniTurri** [google/model-viewer#2818] [Q&A] [answered]
### Description

On examples like https://immersive-web.github.io/webxr-samples/immersive-ar-session.html or https://immersive-web.github.io/webxr-samples/hit-test.html is possible to add additional models using something like:
`let model = new Gltf2Node({url: 'media/gltf/model.gltf'});
      arObject.addNode(model);`

I tried to integrate the two examples by WebXR using the scene Symbol (as found in other issues here), but had no results when opening the model-viewer in ar-modes="xrweb".

I used 
`const modelViewer = document.querySelector('model-viewer');
const $scene = Object.getOwnPropertySymbols(modelViewer)
     .filter(symbol => symbol.description === 'scene')[0];
const scene = modelViewer[$scene]; // is a Three.js Scene instance that contains the model
`

Is it impossible to use those WebXR examples (that integrate ThreeJS) or is my code strategy not right?
Do you have suggestions?

Thank you

#### Live Demo
testino.d3frywawc3uvs7.amplifyapp.com

### Version

- model-viewer: v1.8.0

### Browser Affected
- [ X] Chrome, version: 93.0.4577.82
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [X ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [X ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment, marked as answer]
The whole point of our library is so that you don't have to know about or touch anything related to WebXR or three.js; it all just already works with a declarative HTML element. If you want to go to a deeper level, you  should probably be  using three.js directly  intead of model-viewer. What are you trying to accomplish?

**Reply by GiovanniTurri** [reply]
Ok, thank you so much!
Yes I thought only that it could be compatible but a stand-alone solution (so out of <model-viewer>) is the only way to display in AR more than one model
